[2023-04-14T20:17:22.037+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.choose_endpoint_Kyiv scheduled__2023-04-14T19:00:00+00:00 [queued]>
[2023-04-14T20:17:22.070+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.choose_endpoint_Kyiv scheduled__2023-04-14T19:00:00+00:00 [queued]>
[2023-04-14T20:17:22.072+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-14T20:17:22.080+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-14T20:17:22.082+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-14T20:17:22.109+0000] {taskinstance.py:1309} INFO - Executing <Task(BranchPythonOperator): choose_endpoint_Kyiv> on 2023-04-14 19:00:00+00:00
[2023-04-14T20:17:22.121+0000] {standard_task_runner.py:55} INFO - Started process 15528 to run task
[2023-04-14T20:17:22.126+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'choose_endpoint_Kyiv', 'scheduled__2023-04-14T19:00:00+00:00', '--job-id', '599', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpxtqbw336']
[2023-04-14T20:17:22.134+0000] {standard_task_runner.py:83} INFO - Job 599: Subtask choose_endpoint_Kyiv
[2023-04-14T20:17:22.333+0000] {task_command.py:389} INFO - Running <TaskInstance: weather_dag.choose_endpoint_Kyiv scheduled__2023-04-14T19:00:00+00:00 [running]> on host ffea11fb791c
[2023-04-14T20:17:22.452+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag
AIRFLOW_CTX_TASK_ID=choose_endpoint_Kyiv
AIRFLOW_CTX_EXECUTION_DATE=2023-04-14T19:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-14T19:00:00+00:00
[2023-04-14T20:17:22.456+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:313: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2023-04-14T20:17:22.461+0000] {python.py:177} INFO - Done. Returned value was: retrieve_historical_weather_data_Kyiv
[2023-04-14T20:17:22.462+0000] {python.py:211} INFO - Branch callable return retrieve_historical_weather_data_Kyiv
[2023-04-14T20:17:22.464+0000] {skipmixin.py:155} INFO - Following branch retrieve_historical_weather_data_Kyiv
[2023-04-14T20:17:22.466+0000] {skipmixin.py:211} INFO - Skipping tasks ['retrieve_weather_data_Kyiv']
[2023-04-14T20:17:22.559+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=choose_endpoint_Kyiv, execution_date=20230414T190000, start_date=20230414T201722, end_date=20230414T201722
[2023-04-14T20:17:22.634+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-14T20:17:22.721+0000] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
